### PR TITLE
Add BackgroundFetchManager documents.

### PR DESCRIPTION
**Pull request by jpmedley**
This is a step in addressing #197

### CODE REVIEW

**Comment by Rumyra** [comment]
I'm having a read through now @jpmedley - apologies for the delay 🙈

File: files/en-us/web/api/backgroundfetchmanager/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - Interface
+  - Reference
+  - BackgroundFetchManager
+  - Experimental
+  - Service Workers
+  - Fetch
+---
+<div>{{draft}}{{DefaultAPISidebar("Background Fetch")}}</div>
+
+<p class="summary">The <strong><code>BackgroundFetchManager</code></strong> interface of the {{domxref('Background Fetch','','',' ')}} API is a map where the keys are background fetch IDs and the values are {{domxref("BackgroundFetchRegistration")}} objects.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/index.html:15]: Should this be Background Fetch API?

```suggest
<p class="summary">The <strong><code>BackgroundFetchManager</code></strong> interface of the {{domxref('Background Fetch API','','',' ')}} is a map where the keys are background fetch IDs and the values are {{domxref("BackgroundFetchRegistration")}} objects.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      sizes: '300x300',
+      src: '/ep-5-icon.png',
+      type: 'image/png',
+    }],
+    downloadTotal: 60 * 1024 * 1024,
+  });
+});</pre>
+
+<pre class="brush: js notranslate">
+my code block</pre>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/index.html:49]:
```suggest
```

Comment (Rumyra) [reply files/en-us/web/api/backgroundfetchmanager/index.html:49]: Does that remove those lines 🤔 - that's what I was aiming for

File: files/en-us/web/api/backgroundfetchmanager/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+<dd>Returns a {{jsxref("Promise")}} that resolves with a {{domxref("BackgroundFetchRegistration")}} object for a supplied array of URLs and {{domxref("Request")}} objects.</dd>
+<dt>{{domxref('BackgroundFetchManager.get','get()')}}</dt>
+<dd>Returns a {{jsxref("Promise")}} that resolves with the {{domxref("BackgroundFetchRegistration")}} associated with the provided <code>id</code> or {{jsxref("undefined")}} if the <code>id</code> is not found.</dd>
+<dt>{{domxref('BackgroundFetchManager.getIDs','getIDs()')}}</dt>
+<dd>Returns the IDs of all registered background fetches.</dd>
+</dl>
+
+<h2 id="Examples">Examples</h2>
+
+<p>The example below shows how to get an instance of {{domxref("BackgroundFetchManager")}} from a {{domxreff("ServiceWorkerRegistration")}} object and calls <code>fetch()</code> to download a video in the background.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/index.html:34]:
```suggest
<p>The example below shows how to get an instance of {{domxref("BackgroundFetchManager")}} from a {{domxref("ServiceWorkerRegistration")}} object and calls <code>fetch()</code> to download a video in the background.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/fetch/index.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+<dd>A developer-defined identifier that can be passed to the other methods to retrieve a {{domxref("backgroundFetchRegistration")}}.</dd>
+<dt>request</dt>
+<dd>A {{domxref("RequestInfo")}} object or an array of such objects.</dd>
+<dt>options</dt>
+<dd>A {{domxref("BackgroundFetchOptions")}} object.</dd>
+</dl>
+
+<h3 id="Returns">Return value</h3>
+
+<p>A {{jsxreff("Promise")}} that resolves with a {{domxref("BackgroundFetchRegistration")}} object.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/fetch/index.html:35]:
```suggest
<p>A {{jsxref("Promise")}} that resolves with a {{domxref("BackgroundFetchRegistration")}} object.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/fetch/index.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+     </td>
+    </tr>
+   </tbody>
+  </table>
+</dd>
+</dl>
+
+<h2 id="Examples">Examples</h2>
+
+<p>The following examples shows how to use `fetch()` to create a {{domxref("BackgroundFetchRegistration")}}. With an active service worker, use the {{domxref('ServiceWorkerRegistration.backgroundFetch')}} property to access the `BackgroundFetchManager` object and call its `fetch()` method.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/fetch/index.html:72]: You know what - I did this just a couple of days ago - one day we will have back ticks and I will be happy 😆 

``` suggest
<p>The following examples shows how to use <code>fetch()</code> to create a {{domxref("BackgroundFetchRegistration")}}. With an active {{domxref('ServiceWorker', 'service worker')}}, use the {{domxref('ServiceWorkerRegistration.backgroundFetch')}} property to access the <code>BackgroundFetchManager</code> object and call its <code>fetch()</code> method.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/get/index.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+<h3 id="Parameters">Parameters</h3>
+
+<dl>
+<dt>id</dt>
+<dd>the ID of a {{domxref("backgroundFetchRegistration")}} defined by calling {{domxref("BackgroundFetchRegistration.fetch","fetch()")}}.</dd>
+</dl>
+
+<h3 id="Returns">Return value</h3>
+
+<p>A {{jsxref("Promise")}} that resolves with resolves with a {{domxref("BackgroundFetchRegistration")}} or {{jsxref("undeffined")}}.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/get/index.html:31]:
```suggest
<p>A {{jsxref("Promise")}} that resolves with a {{domxref("BackgroundFetchRegistration")}} or {{jsxref("undefined")}}.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/get/index.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+<h2 id="Examples">Examples</h2>
+
+<p>The following examples shows how to use `get()` to retrieve a {{domxref("BackgroundFetchRegistration")}}. With an active service worker, use the {{domxref('ServiceWorkerRegistration.backgroundFetch')}} to access the `BackgroundFetchManager` object and call its `get()` method.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/get/index.html:35]:
```suggest
<p>The following examples shows how to use <code>get()</code> to retrieve a {{domxref("BackgroundFetchRegistration")}}. With an active {{domxref('ServiceWorker', 'service worker')}}, use the {{domxref('ServiceWorkerRegistration.backgroundFetch')}} to access the <code>BackgroundFetchManager</code> object and call its <code>get()</code> method.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/get/index.html [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+ </tbody>
+</table>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
+
+<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
+
+<p>{{Compat("api.BackgroundFetchManager.get")}}</p>
+
+false
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/get/index.html:65]: Remove this line

```suggest
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+<p class="summary">The <strong><code>getIds()</code></strong> method of the {{domxref("BackgroundFetchManager")}} interface returns the IDs of all registered background fetches. </p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox notranslate">var <var>stringArray</var> = BackgroundFetchManager.getIds();</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+None.
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.html:24]:
```suggest
<p>None.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+<h3 id="Returns">Return value</h3>
+
+<p>A {{jsxref("Promise")}} that resolves with an array off strings.</p>
+
+<h3 id="Exceptions">Exceptions</h3>
+
+None.
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.html:32]:
```suggest
<p>None.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+<pre class="syntaxbox notranslate">var <var>stringArray</var> = BackgroundFetchManager.getIds();</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+None.
+
+<h3 id="Returns">Return value</h3>
+
+<p>A {{jsxref("Promise")}} that resolves with an array off strings.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.html:28]:
```suggest
<p>A {{jsxref("Promise")}} that resolves with an {{jsxref('Array')}} of {{jsxref('String', 'strings')}}.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+<h3 id="Exceptions">Exceptions</h3>
+
+None.
+
+<h2 id="Examples">Examples</h2>
+
+<p>The following examples shows how to retrieve the IDs of all registered background fetches. With an active service worker, use the {{domxref('ServiceWorkerRegistration.backgroundFetch')}} property to access the `BackgroundFetchManager` object and call its `get()` method.</p>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.html:36]:
```suggest
<p>The following example shows how to retrieve the IDs of all registered background fetches. With an active {{domxref('ServiceWorker', 'service worker')}}, use the {{domxref('ServiceWorkerRegistration.backgroundFetch')}} property to access the <code>BackgroundFetchManager</code> object and call its <code>get()</code> method.</p>
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+  const ids = await swReg.backgroundFetch.getIds();
+});</pre>
+
+<p>And/or include a list of links to useful code samples that live elsewhere:</p>
+
+<ul>
+	<li>x</li>
+	<li>y</li>
+	<li>z</li>
+</ul>
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.html:48]: Remove

```suggest
```

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+ </tbody>
+</table>
+
+<h2 id="Browser_compatibility">Browser compatibility</h2>
+
+<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
+
+<p>{{Compat("api.BackgroundFetchManager.getIds")}}</p>
+
+false
```
Comment (Rumyra) [inline files/en-us/web/api/backgroundfetchmanager/getids/index.html:73]: Remove

```suggest
```

**Review by Rumyra — CHANGES_REQUESTED** [review]
Reviewed 🙌

File: files/en-us/web/api/backgroundfetchmanager/fetch/index.html (hunk 4 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/fetch/index.html:35]: I'm not spotting the difference between what I have and what you suggested.

File: files/en-us/web/api/backgroundfetchmanager/fetch/index.html (hunk 5 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/fetch/index.html:72]: Done.

File: files/en-us/web/api/backgroundfetchmanager/get/index.html (hunk 6 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/get/index.html:31]: Fixed.

File: files/en-us/web/api/backgroundfetchmanager/get/index.html (hunk 7 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/get/index.html:35]: Fixed.

File: files/en-us/web/api/backgroundfetchmanager/get/index.html (hunk 8 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/get/index.html:65]: Done.

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html (hunk 11 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/getids/index.html:28]: Done.

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html (hunk 12 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/getids/index.html:36]: Done.

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html (hunk 13 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/getids/index.html:48]: Done.

File: files/en-us/web/api/backgroundfetchmanager/getids/index.html (hunk 14 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/getids/index.html:73]: Done.

File: files/en-us/web/api/backgroundfetchmanager/index.html (hunk 1 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/index.html:15]: Fixed.

File: files/en-us/web/api/backgroundfetchmanager/fetch/index.html (hunk 4 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/fetch/index.html:35]: Fixed.

File: files/en-us/web/api/backgroundfetchmanager/index.html (hunk 3 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/index.html:34]: Fixed.

File: files/en-us/web/api/backgroundfetchmanager/index.html (hunk 2 above)
Comment (jpmedley) [reply files/en-us/web/api/backgroundfetchmanager/index.html:49]: The suggestion button is here:
image

**Comment by jpmedley** [comment]
>I'm having a read through now @jpmedley - apologies for the delay 🙈

I'm not really in a place to criticize right now.

Please take another look.

File: files/en-us/web/api/backgroundfetchmanager/index.html (hunk 2 above)
Comment (Rumyra) [reply files/en-us/web/api/backgroundfetchmanager/index.html]: I've been using the syntax - but probably the button may have helped here

**Review by Rumyra — APPROVED** [review]

**Pull request merged**